monitor/grafana/zeebe.json

```diff
@@ -9539,16 +9539,17 @@
               "fields": "",
               "values": false
             },
-            "showUnfilled": true
+            "showUnfilled": true,
+            "text": {}
           },
-          "pluginVersion": "7.0.3",
+          "pluginVersion": "7.4.5",
           "repeat": "pod",
           "repeatDirection": "h",
           "scopedVars": {
             "pod": {
               "selected": false,
-              "text": "medic-cw-20-f8b1a83e1-benchmark-zeebe-0",
-              "value": "medic-cw-20-f8b1a83e1-benchmark-zeebe-0"
+              "text": "dd-leader-latency-zeebe-0",
```

Comment:
I think next time would be better if you can reset the namespace variable :)

Reply:
Well, I tried to do that by recreating the panel on a fresh dashboard. But the diff was much bigger. No idea why. Grafana dashboard is a mystery :smile: